### ISSUE TITLE
Fetal fixed extremities

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7427]
See
https://api.imrpress.com/sci/redirect/article/pdf?journal=CEOG&volumeCode=47&issueCode=6&doi=10.31083/j.ceog.2020.06.2096
Is this distinct from existing terms to describe lack of fetal movement of extremities?

**Comment by pnrobinson** [comment]
Adding new term: 
- ID: HP:6000770
- label: Fixed fetal extremities
- parents: [Abnormal fetal physiology (HP:0034059)](https://hpo.jax.org/app/browse/term/HP:0034059)
- pmid: 
- def: A condition in which the limbs of a fetus are immobile or held in a rigid position. The limbs are consistently immobile and rigid. Fixed extremities can be observed during prenatal imaging (e.g., ultrasound).
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/7427
- synonyms:
